### PR TITLE
Add ctr model using Criteo dataset

### PR DESCRIPTION
**Pull request by jacquesqiao**

### CODE REVIEW

File: fluid/recommendation/ctr/network_conf.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    dense_input = fluid.layers.data(
+        name="dense_input", shape=[dense_feature_dim], dtype='float32')
+    sparse_input_ids = [
+        fluid.layers.data(
+            name="C" + str(i), shape=[1], lod_level=1, dtype='int64')
+        for i in range(1, 27)
+    ]
+
+    def embedding_layer(input):
+        return fluid.layers.embedding(
```
Comment (guru4elephant) [inline fluid/recommendation/ctr/network_conf.py:18]: 不需要增加is_sparse=True么

File: fluid/recommendation/ctr/preprocess.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+
+@click.command("preprocess")
+@click.option("--datadir", type=str, help="Path to raw criteo dataset")
+@click.option("--outdir", type=str, help="Path to save the processed data")
+def preprocess(datadir, outdir):
+    """
+    All the 13 integer features are normalzied to continous values and these
+    continous features are combined into one vecotr with dimension 13.
+
```
Comment (guru4elephant) [inline fluid/recommendation/ctr/preprocess.py:94]: All 13 integer features are normalized to continuous values and these continuous features are combined into one vector with dimension of 13.

File: fluid/recommendation/ctr/network_conf.py (hunk 1 above)
Comment (jacquesqiao) [reply fluid/recommendation/ctr/network_conf.py:18]: ok, 我加上

**Review by guru4elephant — APPROVED** [review]
LGTM

**Pull request merged**